picky-asn1-x509/src/private_key_info.rs

```diff
@@ -104,24 +112,46 @@ impl<'de> de::Deserialize<'de> for PrivateKeyInfo {
                 A: de::SeqAccess<'de>,
             {
                 let version = seq_next_element!(seq, PrivateKeyInfo, "version");
-                if version != 0 {
-                    return Err(serde_invalid_value!(
-                        PrivateKeyInfo,
-                        "unsupported version (valid version number: 0)",
-                        "a supported PrivateKeyInfo"
-                    ));
-                }
-
-                let private_key_algorithm: AlgorithmIdentifier =
-                    seq_next_element!(seq, PrivateKeyInfo, "private key algorithm");
-                let private_key = if private_key_algorithm.is_a(oids::rsa_encryption()) {
-                    PrivateKeyValue::RSA(seq_next_element!(seq, PrivateKeyInfo, "rsa oid"))
+
+                let tag_peek: TagPeeker = seq.next_element::<TagPeeker>()?.expect("I hope TagPeeker never panics");
```

Comment:
I think there is probably no panic with current code, but what about using the `seq_next_element!` macro like it's done at the other places? Just in case `TagPeeker` deserialization code gets changed.